CMakeLists.txt

```diff
@@ -581,6 +583,10 @@ target_include_directories(ocos_operators PUBLIC
   ${PROJECT_SOURCE_DIR}/base
   ${PROJECT_SOURCE_DIR}/operators)
 
+if (OCOS_USE_CUDA) 
+  target_include_directories(ocos_operators PUBLIC ${cutlass_SOURCE_DIR}/include ${cutlass_SOURCE_DIR}/examples)
```

Comment:
interesting, why does the examples are needs here?

Reply:
some headers are in the example folder, same as ORT. Not sure why